deploy/helm/distributed-compute-operator/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if .Values.podSecurityPolicy.enabled }}
+{{- $elevatePermissions := and .Values.istio.enabled (not .Values.istio.cniPluginInstalled) -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ include "common.names.fullname" . }}
```

Comment:
Just so I sound like a broken record, but just keep in mind this is one of those resources that should include the Helm install's namespace in the name so that multiple installs don't conflict (if it's even possible to install multiple DCOs in multiple namespaces).

(clusterrole too https://github.com/dominodatalab/distributed-compute-operator/pull/16#discussion_r594808462)

Reply:
I must be living in a _Helm v2_ world where namespaced releases didn't exist. I just didn't click instinctively.

If someone creates a release named `steve-o` in two separate namespaces, this and the clusterrole/binding will expose. Thanks for the catch.